examples/common-switches.mk

```diff
@@ -209,6 +209,10 @@ ifeq ($(DISABLE_BUILTIN_MBEDTLS),1)
 configure_OPTIONS              += --disable-builtin-mbedtls
 endif
 
+ifdef BUILTIN_MBEDTLS_MANAGEMENT
```

Comment:
is it possible to use the same style as other flags?

Reply:
The goal with using `ifdef` is to allow the definition for `DOPENTHREAD_CONFIG_ENABLE_BUILTIN_MBEDTLS_MANAGEMENT` to be overridden either with zero or non-zero.  Most other flags are less flexible and only allow setting the value to 1.

I now see another flag with similar behavior, and I could use similar `ifneq` logic for more style consistency:

```
ifneq ($(LOG_OUTPUT),)
COMMONCFLAGS                   += -DOPENTHREAD_CONFIG_LOG_OUTPUT=OPENTHREAD_CONFIG_LOG_OUTPUT_$(LOG_OUTPUT)
endif
```